tests/RobotFramework/tests/pkcs11/private_key_storage.robot

```diff
@@ -82,7 +82,7 @@ Remove Existing Certificates
 
 Set tedge-p11-server Uri
     [Arguments]    ${value}
-    Execute Command    tedge config set device.cryptoki.uri '${value}'
+    Execute Command    tedge config set device.key_uri '${value}'
```

Comment:
Has `device.cryptoki.uri` now been renamed to `device.cryptoki.key_uri`?

But either way we should be using the `device.cryptoki.*` key here as we're talking about configuring the tedge-p11-service and not the client.

Reply:
fixed (rebased)